### PR TITLE
feat(classifier): implement transformer torch sequence classifier

### PR DESCRIPTION
**Pull request by mezig351**
https://github.com/jina-ai/jina-hub/issues/6191 implemented.

### CODE REVIEW

File: classifiers/nlp/TransformersTorchSeqClassifier/tests/test_transformerstorchseqclassifier.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                ]
+            )
+        ),
+    ],
+)
+def test_multilingual(data):
+    model_name = 'nlptown/bert-base-multilingual-uncased-sentiment'
+    classifier = TransformersTorchSeqClassifier(model_name)
+    output = classifier.predict(data)
+    print(output.argmax(axis=1))
```
Comment (nan-wang) [inline classifiers/nlp/TransformersTorchSeqClassifier/tests/test_transformerstorchseqclassifier.py:49]:
```suggestion
```

File: classifiers/nlp/TransformersTorchSeqClassifier/tests/test_transformerstorchseqclassifier.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    assert output.shape[0] == 3
```
Comment (nan-wang) [inline classifiers/nlp/TransformersTorchSeqClassifier/tests/test_transformerstorchseqclassifier.py:50]: Try to make the test easier to understand
```suggestion
    assert output.shape[0] == data.shape[0]
```

**Review by nan-wang — CHANGES_REQUESTED** [review]

**Comment by nan-wang** [comment]
Good job! Many thanks for your contribution.

**Comment by nan-wang** [comment]
recheckcla

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Comment by nan-wang** [comment]
**Jina CLA check**<br/>
  ❤️ Thank you for your pull request. It looks like this is your first contribution to an open source project maintained by Jina AI Limited. Before we can look at your pull request, we kindly ask that you sign our [Contributor License Agreement](https://github.com/jina-ai/jina/blob/master/.github/CLA.md). You can sign it by commenting in format below.
  - - -
  ***I have read the CLA Document and I hereby sign the CLA***
  - - -

**Comment by nan-wang** [comment]
hi, there. @mezig351  Thanks for the PR. Your contribution is greatly appreciated. Would you please sign the CLA above so that we can proceed to merge this PR?

**Comment by mezig351** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**